cli/cli.go

```diff
@@ -25,6 +26,7 @@ var (
 	dumpPem      = dump.Flag("pem", "Write output as PEM blocks instead of human-readable format.").Short('m').Bool()
 	dumpJSON     = dump.Flag("json", "Write output as machine-readable JSON format.").Short('j').Bool()
 	dumpFirst    = dump.Flag("first", "Only display the first certificate. This flag can be paired with --json or --pem.").Short('l').Bool()
+	dumpSha256   = dump.Flag("sha256", "Write sha256sum of cert to output, not compatible --pem or --json.").Short('s').Bool()
```

Comment:
Any reason this fingerprint couldn't also be added in JSON output? 

And I wonder if we could name that something like --show-fingerprint  (not sure what a good one letter flag to give it then, though) and just default to sha-256 for now? If we ever wanted to provide other hashing means we could turn the flag into an option with an argument.

Reply:
It definitely could be added, but look later how if *dumpJson {} is implemented. I'd have to rework the JSON printing and didn't think it was worth it yet.